app/src/androidTest/java/io/novafoundation/nova/MoonbaseSendIntagrationTest.kt

```diff
@@ -50,13 +55,37 @@ class MoonbaseSendIntagrationTest {
         return EthereumKeypairFactory.generate(seed, junctions)
     }
 
+    private inner class TestSigner(
+        private val testPayload: (SignerPayloadExtrinsic) -> Unit
+    ) : NovaSigner {
+
+        val accountId = ByteArray(32) { 1 }
```

Comment:
It seems we don't use it but to be semantically correct let's provide accountId to the constructor from outside